email/sqr-submission.html

```diff
@@ -0,0 +1,76 @@
+<body>
+  <div style="color:red; font-weight:bold;">
+    {% if params.submission.cancellation.notPreferredLanguage %}
+      <p>The lecture is not in devotee's preferred language!</p>
+    {% endif %}
+
+    {% if params.submission.cancellation.audioProblem %}
+      <p>Unable to play or download the audio {{ params.submission.comments }}</p>
+    {% endif %}
+
+    {% if params.submission.changed != params.submission.completed %}
+      <p>This is an updated submission!</p>
+    {% endif %}
+
+    {% if params.fileData.soundQualityReporting.assignee.emailAddress != params.submission.author.emailAddress %}
```

Comment:
This condition is related only to the first warning in the following block. Other warnings should have its own condition. So no need in `div` here, `p` is enough.